opengrok-indexer/src/main/java/org/opengrok/indexer/web/messages/MessagesUtils.java

```diff
@@ -230,4 +234,32 @@ private static String messagesToJson(Group group, String... additionalTags) {
     public static String messagesToJson(Group group) {
         return messagesToJson(group, new String[0]);
     }
+
+    /**
+     * @return name of highest cssClass of messages present in the system or null.
+     */
+    static String getHighestMessageLevel(Collection<MessagesContainer.AcceptedMessage> messages) {
+        return messages.
+                stream().
+                map(MessagesContainer.AcceptedMessage::getMessageLevel).
+                max(VALUE_COMPARATOR).
```

Comment:
I'd qualify here with the class name so the name does not seem a little out of context.